styx-api-service/src/test/java/com/spotify/styx/api/BackfillResourceTest.java

```diff
@@ -917,6 +933,28 @@ public void shouldHaltBackfill() throws Exception {
     verify(serviceHelper.stubClient(), times(1)).send(any());
   }
 
+  @Test
+  public void shouldHaltBackfillUpdateLastModified() throws Exception {
```

Comment:
We can perhaps refactor existing test cases to include timestamp assertion than adding new ones. Wdyt?